packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard/signup-wizard.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import { Wizard } from '@deriv/ui';
+
+type TSignupWizardProps = {
+    setIsWizardOpen: (is_wizard_open: boolean) => void;
+};
+
+const SignupWizard = ({ setIsWizardOpen }: TSignupWizardProps) => {
+    const [create_app_state, setCreateAppState] = React.useState({});
```

Comment:
```suggestion
```
@george-usynin-binary It's unused 🤔

Reply:
@farzin-deriv for now yes, i have made a mocked wizard, but i better remove it, to not confuse